routes/api.php

```diff
@@ -123,6 +124,12 @@
             Route::get('follow-requests', [FollowController::class, 'getFollowRequests']);
             Route::get('followings', [FollowController::class, 'getFollowings']);
         });
+        Route::group(['prefix' => 'webhooks'], static function() {
+            Route::get('/', [WebhookController::class, 'getWebhooks']);
+            Route::post('/', [WebhookController::class, 'createWebhook']);
+            Route::delete('/{webhookId}', [WebhookController::class, 'deleteWebhook']);
+            Route::delete('/', [WebhookController::class, 'deleteAllWebhooks']);
```

Comment:
I suspect a `deleteAllWebhooks` route to be a footgun, especially if it isn't specified with an `all` keyword in the URL, but just by leaving the ID blank (although I understand that this is how REST works).

Why would someone use this endpoint? Please help me understand your thought process here :)

Reply:
It was useful for testing it locally, but yeah it's probably a footgun and we should remove it